src/datachain/lib/func/func.py

```diff
@@ -9,18 +11,44 @@
     from datachain.lib.signal_schema import SignalSchema
 
 
+class Window:
+    def __init__(self, *, partition_by: str, order_by: str, desc: bool = False) -> None:
+        self.partition_by = partition_by
```

Comment:
do we cover a scenario like this with this:

```
I need to select a subset of the records with 5 records in each class.
```


?

Reply:
Yes, added test for this scenario: https://github.com/iterative/datachain/pull/515/files#diff-b40ca373e4b7558183b740420a85bc3d95c55a4b3049addcbbc1cf0b04460eecR1421-R1463

Reply:
I'm not sure we need a test for this tbh (if it doesn't cover any additional window functionality it becomes a functional test / not a unit one, not sure if there is value in this - your call).

What I mean is that we might want to have a simpler syntax for this? It seems that tools like IBIS have `range` that can be passed inside `window`. Can we explore that option? Would it be solving this type of scenarios in a simpler way? wdyt?

Reply:
Got it. Sounds like a good idea to me.
May be in follow-up issue (I can create one)?

Reply:
Created a follow-up issue: https://github.com/iterative/datachain/issues/522

Reply:
thanks!